src/indicadores/infrastructure/repositories.py

```diff
@@ -43,6 +43,20 @@ def get_last_formula_id(self, formula_id:str, tipo_identificacion:str, identific
             return Indicador(
                 valor= float(0)
             )
+    
+    def get_last_indicador(self, formula_id:str, tipo_identificacion:str, identificacion:str, sesionId:str) -> Indicador:
+        try:
+            indicador_dto = (
+                    db.session.query(IndicadorDTO)
+                    .filter_by(tipo_identificacion=tipo_identificacion)
+                    .filter_by(identificacion=identificacion)
+                    .filter_by(idFormula=formula_id)
+                    .filter_by(idSesion=sesionId)
+                    .one()
```

Comment:
No se obtiene el último indicador. Este query solamente obtiene un indicador, no el último. Falta ordenar for fecha